experimental/auth-logic/wrappers/rekor_wrapper.go

```diff
@@ -185,10 +186,22 @@ func checkEntryPubKeyMatchesExpectedKey(rekordEntry *rekord.V001Entry, prodTeamK
 	return nil
 }
 
+// compareEndorsementAndRekordHash compares the hash referenced in a
+// rekord entry to the hash of input bytes.  For this use-case, the input
+// bytes will be an endorsement file
+func compareEndorsementAndRekordHash(rekordEntry *rekord.V001Entry, endorsementBytes []byte) error {
```

Comment:
```suggestion
func compareEndorsementAndRekorHash(rekordEntry *rekord.V001Entry, endorsementBytes []byte) error {
```

nit: I think we decided to user rekor.